packages/aws-cdk-lib/aws-route53/lib/health-check.ts

```diff
@@ -0,0 +1,517 @@
+import { Construct } from 'constructs';
+import { CfnHealthCheck } from './route53.generated';
+import { Duration, IResource, Resource } from '../../core';
+
+/**
+ * Imported or created health check
+ */
+export interface IHealthCheck extends IResource {
+  /**
+   * The ID of the health check.
+   *
+   * @attribute
+   */
+  readonly healthCheckId: string;
+}
+
+/**
+ * Properties for a new health check.
+ */
+export interface HealthCheckProps {
+  /**
+   * The type of health check to be associated with the record.
+   */
+  readonly type: HealthCheckType;
+
+  /**
+   * CloudWatch alarm that you want Amazon Route 53 health checkers to use to determine whether the specified health check is healthy.
+   *
+   * @default - no alarm identifier
+   */
+  readonly alarmIdentifier?: AlarmIdentifier;
+
+  /**
+   * A complex type that contains one ChildHealthCheck element for each health check that you want to associate with a CALCULATED health check.
+   *
+   * @default - no child health checks
+   */
+  readonly childHealthChecks?: string[];
+
+  /**
+   * Specify whether you want Amazon Route 53 to send the value of FullyQualifiedDomainName to the endpoint in the client_hello message during TLS negotiation. This allows the endpoint to respond to HTTPS health check requests with the applicable SSL/TLS certificate.
+   *
+   * @default - not configured
```

Comment:
I strongly suspect the default is `false` here, but this is not made clear in either the Cfn nor the API documentation

Reply:
Again, I don't think setting a default value in the CDK is that useful